Lesson25/Anil/map.js

```diff
@@ -0,0 +1,347 @@
+// Exercise 1:
+// Given an array of strings, convert each string to uppercase using .map().
+const wordsExercise1 = [
+	'apple',
+	'banana',
+	'car',
+	'dog',
+	'elephant',
+	'flower',
+	'guitar',
+	'house',
+	'island',
+	'jungle',
+];
+
+const upperCaseStrings = [...wordsExercise1].map((x)=>x.toUpperCase());
```

Comment:
You don't need to copy initial array when using map.
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/map

Reply:
Also, naming is important:
```suggestion
const upperCaseStrings = wordsExercise1.map((string)=>string.toUpperCase());
```